generators/entity/index.js

```diff
@@ -1000,12 +1020,53 @@ module.exports = EntityGenerator.extend({
 
     prompting: {
         /* pre entity hook needs to be written here */
+        askForMicroserviceJson: function(){
+            if(this.applicationType != 'gateway') {
+                return;
+            }
+
+            var cb = this.async();
+
+            var prompts = [
+                {
+                    type: 'confirm',
+                    name: 'useMicroserviceJson',
+                    message: 'Do you want to generate this entity from an existing microservice?',
+                    default: true
+                },
+                {
+                    when: function(response) {
+                        return response.useMicroserviceJson == true;
+                    },
+                    type: 'input',
+                    name: 'microservicePath',
+                    message: 'Enter the absolute path to the microservice root directory:',
+                    validate: function(input) {
+                        var fromPath = input + '/' + this.filename;
+                        if (shelljs.test('-f', fromPath)) {
+                            return true;
+                        }
+                        else {
+                            return this.filename + ' not found in ' + input + '/';
+                        }
+                    }.bind(this)
+                }
+            ];
+
+            this.prompt(prompts, function(props) {
+                this.log(chalk.green('\nFound the ' + this.filename + ' configuration file, entity can be automatically generated!\n'));
+                this.microservicePath = props.microservicePath;
+                this.importMicroserviceJson = true;
```

Comment:
But if we decide to use a `remote-entities` JSON file should I still copy the json in `.jhipster` folder ?